operating/examine-images-for-vulnerabilities.adoc

```diff
@@ -47,6 +47,181 @@ It may fail to identify installed components or vulnerabilities if the final ima
+----
++
+For more information about setting the environment variables, see xref:../installing/installing_ocp/install-secured-cluster-ocp.adoc#secured-cluster-services-environment-variables_install-secured-cluster-ocp[Environment variables].
+
+
+[id="configuring-environment-variables-after-installation_{context}"]
+==== Configuring environment variables after installation
+
+[discrete]
+===== OpenShift Container Platform
+
+.Procedure
+
+* Run the following command to enable delegated scanning by setting the value of the environment variable`ROX_DELEGATED_SCANNING_DISABLED` to `true` for a deployment named `sensor`:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc set env deploy/sensor ROX_DELEGATED_SCANNING_DISABLED="true"
+----
+
+[discrete]
+===== Kubernetes
+
+.Procedure
+
+* Run the following command to enable delegated scanning by setting the value of the environment variable `ROX_DELEGATED_SCANNING_DISABLED` to `true` for a deployment named `sensor`:
++
+[source,terminal,subs="+quotes"]
+----
+$ kubectl set env deploy/sensor ROX_DELEGATED_SCANNING_DISABLED="true"
+----
+
+Sensor should restart automatically. However, if it doesn't restart on its own, you should manually cause it to restart for the changes to take effect.
+
+If you're using the {product-title-short} Operator, there is a possibility that this change can be undone. In such a case, you should follow the specific instructions of the Operator.
+
+
+[id="configuring-delegated-image-scanning_{context}"]
+=== Configuring delegated image scanning
+
+A new delegated registration configuration specifies the registries from which image queries are to be delegated. For images observed by Sensor, this configuration allows you to delegate scans from no registries (default), all registries, or specific registries. To enable delegation of scans using the `roxctl` CLI, the Jenkins plugin, or the API, you must also specify a destination cluster and source registry.
+
+
+.Prerequisites
+
+* The environment variables must be configured. For more information on how to configure environment variables, see xref:../operating/examine-images-for-vulnerabilities.adoc#configuring-environment-variables_examine-images-for-vulnerabilities[Configuring environment variables].
+
+.Procedure
+
+. In the {product-title-short} portal, navigate to *Platform Configuration -> Clusters*.
+. In the *Clusters* view header, click *Manage delegated scanning*.
+. In the *Delegated Image Scanning* page, provide the necessary information:
+** *Delegate scanning for*: Choose the scope of image delegation. Select one of the following options:
+*** None (default)
+*** All registries
+*** Specified registries
+** *Select default cluster to delegate to*: From the drop-down list, select the name of the destination cluster to which the delegated scans should be sent.
+** Click *Add registry*. Specify the source registry and destination cluster details. You can add more than one registry if required.
```

Comment:
@dcaravel 

Likewise, the way this is worded, it sounds like adding at least one row to this list is required, but you said it's not.

We should specify when you would want to add one or more lines here, and why you might leave the cluster portion of this set to none.

Reply:
Agreed, see my response above. Believe adding more details on these settings and how sensor reacts separatly from how roxctl/API requests are handled is needed.